doc/release-notes.md

```diff
@@ -57,9 +43,9 @@ built using Qt 5.9.x, which doesn't support versions of macOS older than
 10.10. Additionally, Bitcoin Core does not yet change appearance when
 macOS "dark mode" is activated.
 
-In addition to previously-supported CPU platforms, this release's
-pre-compiled distribution also provides binaries for the RISC-V
-platform.
+Users running macOS Catalina may need to "right-click" and then choose "Open"
+to open the Bitcoin Core .dmg. This is due to new signing requirements
+imposed by Apple, which the Bitcoin Core project does not yet adhere too.
 
 Notable changes
```

Comment:
Shouldn't we mention #16714 here?
It is a really notable change for bitcoin-qt users.

Reply:
Gui users will either see the intro screen and the new feature, because it is a fresh install. Or they don't, because it is not a fresh install. In either case it wouldn't help them if they knew about it upfront. Also, at some point we need to pull the release notes and push out the release.